docs/setup-dev-environment.md

```diff
@@ -57,129 +130,282 @@ EOF
+Download the test data.
 
+```bash
+python get_test_polygons.py
 ```
-sudo su root
-export NODEJS_HOME=/usr/local/lib/node/node-v12.14.1/bin/
-export PATH=$NODEJS_HOME:$PATH
-npm install -g npm@next
-exit
+
+Note: the first time you run it, you will get these warnings:
+```
+rm: cannot remove 'test_buildings.geojson': No such file or directory
+rm: cannot remove 'test_buildings.3857.csv': No such file or directory
 ```
 
-Now install the required Node packages. This needs to done from the `app` directory of your
-local repository, so that it can read from the `package.json` file.
+#### Run migrations
 
-`cd ./colouring-london/app && npm install`
+Now run all 'up' migrations to create tables, data types, indexes etc. The `.sql` scripts to
+do this are located in the `migrations` folder of your local repository.
 
+```bash
+ls ~/colouring-london/migrations/*.up.sql 2>/dev/null | while read -r migration; do psql -d <colouringlondondb> < $migration; done;
+```
 
-## Running the application
+#### Load buildings
 
-Now we are ready to run the application. The `APP_COOKIE_SECRET` is arbitrary.
+Load all building outlines.
 
-`PGPASSWORD=<pgpassword> PGDATABASE=<colouringlondondb> PGUSER=<username> PGHOST=localhost PGPORT=5432 APP_COOKIE_SECRET=123456 npm start`
+```bash
+./load_geometries.sh ./
+```
 
-If you a running Ubuntu in a virtual environment you will need to configure networking to
-forward ports from the guest to the host. For Virtual Box the following was configured under
-NAT port forwarding.
+Create a building record per outline.
 
-Name     | Protocol  | Host Port  | Guest Port
--------- | --------- | ---------- | -----------
-app      | TCP       | 8080       | 3000
-app_dev  | TCP       | 3001       | 3001
-ssh      | TCP       | 4022       | 22
+```bash
+./create_building_records.sh
+```
+</details>
+
+## :computer: Running the application
+
+Now we are ready to run the application.
+
+First enter the app directory.
+
+```bash
+cd ~/colouring-london/app
+```
+
+Then create a folder for the tilecache.
+
+```bash
+mkdir tilecache
+```
+
+Create some additional variables for running the application (the `APP_COOKIE_SECRET` is arbitrary).
+
+```bash
+export PGPORT=5432
+export APP_COOKIE_SECRET=123456
+export TILECACHE_PATH=~/colouring-london/app/tilecache
+```
+
+Finally, simply run the application with npm.
+
+```bash
+npm start
+```
+
+**Note:** You can also specify the variables for `npm start` like so:
+<details>
+<summary>
+Specify variables
+</summary>
+
+```bash
+PGPASSWORD=<pgpassword> PGDATABASE=<colouringlondondb> PGUSER=<username> PGHOST=localhost PGPORT=5432 APP_COOKIE_SECRET=123456 TILECACHE_PATH=~/colouring-london/app/tilecache npm start
+```
+
+</details><p></p>
+
+### :eyes: Viewing the application
 
-The site can then be viewed on http://localhost:8080. The `app_dev` mapping is used in
-development by Razzle which rebuilds and serves client side assets on the fly.
+The site can then be viewed on http://localhost:8080.
```

Comment:
```suggestion
The site can then be viewed on http://localhost:8080 on the host computer.
```

clarify where it can be seen (not tested, based on config listed above)